ddmrp_chatter/readme/HISTORY.rst

```diff
@@ -0,0 +1,4 @@
+13.0.1.0.0 (2020-09-10)
```

Comment:
Can you add a new "history entry here" saying that the module has been migrated to v14?

Reply:
Done!